File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 2/src/Ex7.java
//Importando Scanner
import java.util.Scanner;

public class Ex7{
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner ent = new Scanner(System.in);
	
	System.out.println("Reajuste Salarial\n\n");
	
	//vars
	double salarioAtual, novoSalario;
	
	//Entrada
	System.out.print("Insira o salário do funcionário: ");
	salarioAtual = ent.nextDouble();
	
	//Processamento
	if ( salarioAtual < 500 )
	{
		novoSalario = salarioAtual * 1.3;
		System.out.printf("Salário com reajuste: R$%.2f", novoSalario);
	}
	else
		System.out.println("O funcinário não possui direito ao reajuste salarial");
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 2/ex7.c
#include <stdio.h>

main()
{
    printf("Reajuste salarial\n\n");

    //vars
    float salarioInicial, salarioFinal;

    //Entrada
    printf("Insira o salario do funcionario: ");
    scanf("%f",&salarioInicial);

    //Processamento
    if ( salarioInicial < 500 )
    {
        salarioFinal = salarioInicial * 1.3;
        printf("Salario com reajuste de 30%: R$%.2f", salarioFinal);
    }
    else
        printf("O funcionario nao possui direito ao aumento de 30%");

}
